apps/src/templates/progress/SendLesson.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import SendLessonDialog from './SendLessonDialog';
+
+export default class SendLesson extends React.Component {
+  static propTypes = {
+    lessonUrl: PropTypes.string.isRequired,
+    lessonTitle: PropTypes.string,
+    courseid: PropTypes.number,
```

Comment:
NIT: is `courseid` supposed to be `courseId` (capital I)?

Reply:
Heh, I originally had it as `courseId` but the Google share button component calls the property `courseid` and I decided that it would be less confusing if it was consistent with that (because this property only applies to the Google scenario).  Knowing that, do you still think I should change it to `courseId`?  (I don't feel strongly about it either way.)

(Another option would be to rename the property in Google share button but I didn't want to touch that in this PR.)

Reply:
Nah, I don't feel strongly about it.